config/locales/cy.yml

```
@@ -99,24 +99,28 @@ cy:
     upcoming_bank_holidays: Gwyliau banc i ddod
   continue:
   cookies:
-    additional:
     always_on:
     clicks:
     confirmation_message:
     confirmation_previous_referrer_link:
     confirmation_title:
+    explaination:
```

```suggestion
    explanation:
```

The view references to this key will need updating too.

---

Sorry about this one.